packages/framework/src/Model/Product/Collection/ProductUrlsBatchLoader.php

```diff
@@ -6,28 +6,24 @@
 
 use Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig;
 use Shopsys\FrameworkBundle\Component\Image\ImageUrlWithSizeHelper;
+use Shopsys\FrameworkBundle\Component\LocalCache\LocalCacheFacade;
 use Shopsys\FrameworkBundle\Model\Product\Collection\Exception\ProductImageUrlNotLoadedException;
 use Shopsys\FrameworkBundle\Model\Product\Collection\Exception\ProductUrlNotLoadedException;
 use Shopsys\FrameworkBundle\Model\Product\Product;
-use Symfony\Contracts\Service\ResetInterface;
 
-class ProductUrlsBatchLoader implements ResetInterface
+class ProductUrlsBatchLoader
 {
-    /**
-     * @var string[]
-     */
-    protected array $loadedProductUrls = [];
-
-    /**
-     * @var string[]|null[]
-     */
-    protected array $loadedProductImageUrls = [];
+    protected const PRODUCT_URLS_CACHE_NAMESPACE = 'loadedProductUrls';
+    protected const PRODUCT_IMAGE_URLS_CACHE_NAMESPACE = 'loadedProductImageUrls';
```

Comment:
```suggestion
    protected const string PRODUCT_URLS_CACHE_NAMESPACE = 'loadedProductUrls';
    protected const string PRODUCT_IMAGE_URLS_CACHE_NAMESPACE = 'loadedProductImageUrls';
```

We should probably add some fixer for this :smile: